src/inject/GPMInject/interface/customUI.js

```diff
@@ -34,7 +34,14 @@ window.wait(() => {
   // Settings options that won't work
   hide('[data-action="upload-music"]');
   hide('[data-action="help-and-feedback"]');
+  hide('[data-action="send-gift"]');
```

Comment:
This is also in another PR (I think yours o_0)

Be careful not to bleed commits / delta into other PR's / branches